### ISSUE TITLE
phi4

### ISSUE THREAD

**Issue by sinxyz** [ollama/ollama#8159]
please add phi4

**Comment by ejgutierrez74** [comment]
ollama run vanilj/Phi-4 ?? https://ollama.com/search?o=newest&q=phi4 ??

**Comment by pdevine** [comment]
Microsoft hasn't released Phi 4 as an Open Source model. We'll support it when they release it.

**Comment by ejgutierrez74** [comment]
And  https://ollama.com/search?o=newest&q=phi4 ?? ??? Is fake ?

**Comment by pdevine** [comment]
> And https://ollama.com/search?o=newest&q=phi4 ?? ??? Is fake ?

@ejgutierrez74 Yeah, it was the leaked weights and it wasn't the final release with the proper safety mechanisms. @MathiasSchindler You can run the latest (real!) model with `ollama run phi4`.
